resources/benchmark-report.mjs

```diff
@@ -1,83 +1,107 @@
 // This file can be customized to report results as needed.
 
 (function () {
-    if (!window.testRunner && location.search != '?webkit' && location.hash != '#webkit')
+    if (
+        !window.testRunner &&
+        location.search !== "?webkit" &&
+        location.hash !== "#webkit"
+    )
         return;
 
-    if (window.testRunner)
-        testRunner.waitUntilDone();
+    if (window.testRunner) window.testRunner.waitUntilDone();
 
-    const scriptElement = document.createElement('script');
-    scriptElement.src = '../resources/runner.js';
+    const scriptElement = document.createElement("script");
+    scriptElement.src = "../resources/runner.js";
     document.head.appendChild(scriptElement);
 
-    const styleElement = document.createElement('style');
-    styleElement.textContent = 'pre { padding-top: 600px; }';
+    const styleElement = document.createElement("style");
+    styleElement.textContent = "pre { padding-top: 600px; }";
     document.head.appendChild(styleElement);
 
     let createTest;
-    const valuesByIteration = new Array;
+    const valuesByIteration = new Array();
 
     window.onload = () => {
-        document.body.removeChild(document.querySelector('main'));
+        document.body.removeChild(document.querySelector("main"));
         startBenchmark();
     };
 
     window.benchmarkClient = {
         iterationCount: 5, // Use 4 different instances of DRT/WTR to run 5 iterations.
         willStartFirstIteration(iterationCount) {
-            createTest = (name, aggregator, isLastTest, unit = 'ms') => {
+            createTest = (name, aggregator, isLastTest, unit = "ms") => {
                 return {
                     customIterationCount: iterationCount,
                     doNotIgnoreInitialRun: true,
                     doNotMeasureMemoryUsage: true,
                     continueTesting: !isLastTest,
                     unit: unit,
                     name: name,
-                    aggregator: aggregator};
-            }
-            PerfTestRunner.prepareToMeasureValuesAsync(createTest(null, 'Geometric'));
+                    aggregator: aggregator,
+                };
+            };
+            PerfTestRunner.prepareToMeasureValuesAsync(
+                createTest(null, "Geometric")
+            );
         },
         didRunSuites(measuredValues) {
             PerfTestRunner.measureValueAsync(measuredValues.geomean);
             valuesByIteration.push(measuredValues);
         },
         didFinishLastIteration() {
-            document.head.removeChild(document.querySelector('style'));
+            document.head.removeChild(document.querySelector("style"));
 
             const measuredValuesByFullName = {};
             function addToMeasuredValue(value, fullName, aggregator) {
-                const values = measuredValuesByFullName[fullName] || new Array;
+                const values =
+                    measuredValuesByFullName[fullName] || new Array();
```

Comment:
What's up with this?
Are we trying to set 80 character limit or something.
We certainly shouldn't.